### ISSUE TITLE
Make NACM default-deny-all and default-deny-write built-in statements

### ISSUE THREAD

**Issue by abierman** [netmod-wg/yang-next#61]
Ready to promote these to language statements so it is not NACM-specific

**Comment by kwatsen** [comment]
Andy: test case, how would we make an extension a real statement
Juergen: if a test case, we definitely should do this
Robert: support doing the test

Kent: would this obsolete NACM RFC?
Andy: that's the issue, how to support both going forward, etc.

**Comment by mbj4668** [comment]
> Kent: would this obsolete NACM RFC?

No, but if we did an update to NACM, the extension would be marked as deprecated or obsolete.

**Comment by rgwilton** [comment]
I think that this issue also raises the general question as to whether it is good modularity to bundle typedefs and extensions into the same YANG module as data nodes that implement the feature.  E.g. perhaps, in hindsight it would have been better to split the NACM yang module into a separate typedef & extension vs implementation?

**Comment by reshad-equinix** [comment]
So if default-deny-all and default-deny-write become built-in, they wouldn't apply just to NETCONF but to the model. LGTM.

**Comment by kwatsen** [comment]
Kent: this regards more the sensitivity/security of nodes.
Andy: the offline-validation problem has been around for awhile.
Jan: running-alone problem may not occur in real world
Andy: because Administrator has full access.
Juergen: not the intent to redefine anything - working solo already (hence importance medium)
Andy: having the keyword built-in would negate the need to import nam just to get the extension statements.
